### PR TITLE
commit

### LINKED ISSUE #16: Fixes #12
Fixes #12

_Originally posted by @jiny-joseph in https://github.com/githubschool/amd-ryzen/issues/8#issuecomment-738299673_

### PR DESCRIPTION
**Pull request by jiny-joseph**
test fixes #16

### CODE REVIEW

**Comment by jiny-joseph** [comment]
Fixes #12

File: _slides/10-jiny-joseph.md [hunk 1]
```diff
@@ -3,7 +3,8 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Hi Jiny Here
```
Comment (DevGuangLi) [inline _slides/10-jiny-joseph.md:6]: looks great

**Review by DevGuangLi — APPROVED** [review]
looks great

File: jiny-joseph-test-git [hunk 2]
```diff
@@ -0,0 +1 @@
+test file
```
Comment (jiny-joseph) [inline jiny-joseph-test-git:1]: test comment

**Review by jiny-joseph — COMMENTED** [review]
test

**Pull request merged**